store/tikv/client_batch.go

```diff
@@ -76,10 +75,12 @@ func (a *batchConn) fetchAllPendingRequests(
 		a.idleDetect.Reset(idleTimeout)
 	case <-a.idleDetect.C:
 		a.idleDetect.Reset(idleTimeout)
-		a.idle = true
-		atomic.CompareAndSwapUint32(a.idleNotify, 0, 1)
-		// This batchConn to be recycled
-		return
+		// eventCh signal is handle each time SendRequest is called, it's unlikely
```

Comment:
can we add some check grpc connection is still alive in here?

it seem current idle check logic is "it's idle if there are no any request this request"...

but there are some situation @qw4990 meet, a system do a batch process every 6 hours, it seems this will close connection even if tikv still alive

Reply:
> a system do a batch process every 6 hours, and current check idle logic will close connection even if tikv still alive

1. the idle mechanism does not close the connection permanently, it works together with an auto-recreate mechanism.
2. it's unlikely that active connection been recycled, our system has background SQL jobs anyway.